### DISCUSSION TITLE
Request Mute or Block users

### DISCUSSION THREAD

**Discussion by triptych** [dailydotdev/daily#514] [Feature Requests]
As you are allowing different users to reply to comments, please allow one  to mute or block another user if they choose not to further engage with that user.

**Comment by idoshamun** [comment]
Hi @triptych,

Thanks for the suggestion. It's definitely something we have to consider along the way.
I'll keep you posted

**Reply by jgarcianewemage** [reply]
it s been 2 years, how long is this way? xD

**Reply by idoshamun** [reply]
It's currently not in development 😔

**Comment by ghost** [comment]
I agree with the need for this feature. There are some users on here where I'd rather not have to interact with nor see their rude, non-constructive comments.

**Comment by jgarcianewemage** [comment]
2024 you still cannot block Aholes and trolls

**Comment by thraizz** [comment]
With the rise of generated spam, this becomes more and more urgent, i'm less inclined to interact with the comments at all if people like this become the majority:
![image](https://github.com/user-attachments/assets/37502af0-590a-43f3-9b4b-eeeccd94c5b3)

**Comment by idoshamun** [comment]
Quick update, everyone: I'm happy to share what's on the backlog, and I hope we can fix it soon.
With that said, I'd like to ask you to report inappropriate comments or AI-generated ones. You can also downvote them.

**Comment by idoshamun** [comment]
Happy to share that it's possible to block and report users now. It's been a while but it's finally here :)
